gammapy/data/data_store.py

```diff
@@ -711,3 +713,59 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_rmf(self, position, e_true, e_reco, low_reco_threshold=None, high_reco_threshold=None):
```

Comment:
You call the method "make_mean_rmf", but return an "EnergyDispersion" object.

My understanding is that "rmf" is "Redistribution Matrix File", a file format:
http://heasarc.gsfc.nasa.gov/docs/heasarc/caldb/docs/memos/cal_gen_92_002/cal_gen_92_002.html#tth_sEc3

So I'd be +1 to use `edisp` for "energy dispersion" in Gammapy, and to only use "RMF" in docs or on file I/O,
i.e. call this method "make_mean_edisp".